spec/factories/curator/filestreams/metadata.rb

```diff
@@ -7,6 +7,16 @@
     file_set_type { 'Curator::Filestreams::Metadata' }
     file_name_base { Faker::Internet.uuid }
     position { 1 }
+    hand_side { 'left' }
+    page_type { 'TOC' }
+    page_label { 'OAI HARVEST META' }
```

Comment:
`hand_side`, `page_type`, and `page_label` would not be used with this FileSet type